README.md

```diff
@@ -137,34 +137,43 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
     </sly>
     ```
     
-  - [4.3](#4.3) <a name='4.3'></a> **Use meaningful identifier names.**
+  - [4.3](#4.3) <a name='4.3'></a> **Use naming conventions for identifiers.**
   
+    Sightly knows the following types of identifiers:
+    * defined in `data-sly-use` for a template library
+    * defined in `data-sly-template` for template names
+    * defined in `data-sly-use` for a JavaScript/Java object variable
+    * defined in `data-sly-list` for a list entry variable
+    * defined in `data-sly-test` for a test result variable
+    
+    To be able to distinguish the former two identifiers, the following naming conventions should be used:
+    * for template libraries: the prefix **lib-** followed by an arbitrary name
+    * for template names: the prefix **template-** followed by an arbitrary name
+    * for all other variable identifiers any name except the ones listed in http://docs.adobe.com/docs/en/aem/6-1/develop/sightly/global-objects.html can be used.
     This will enhance the readability of your Sightly scripts and and makes it easier for others to understand.
 
     ```html
-    <!--/* Bad */-->
-    <sly data-sly-use.comp="com.example.TeaserComponent">
-        ...
-    </sly>
-     
-    <!--/* Good */-->
-    <sly data-sly-use.teaser="com.example.TeaserComponent">
-        ...
+    <!--/* Java-object and template library */-->
+    <sly data-sly-use.teaser="com.example.TeaserComponent" data-sly-use.lib-teaser="teaser-templates.html">
+    	<!--/* template name> */-->
+        <sly data-sly-template.template-teaser1>...</sly>
+        <sly data-sly-call="template-teaser1"></sly>
+        <sly data-sly-call="lib-teaser.teaser3"></sly>
     </sly>
```

Comment:
The main issue you are addressing is to distinguish between template libraries and template names. Couldn't this be solved by just using better identifier names? Something like this:

```html
<sly data-sly-use.teaser="com.example.TeaserComponent" data-sly-use.teaserTemplates="teaser-templates.html">
    <sly data-sly-template.teaserTemplateSmall>
        ...
    </sly>

    <sly data-sly-call="teaserTemplateSmall"></sly>
    <sly data-sly-call="teaserTemplates.teaserTemplateMedium"></sly>
</sly>
```

What do you think?